src/sdk/pynni/nni/compression/torch/quantization/quantizers.py

```diff
@@ -110,12 +112,111 @@ def get_bits_length(config, quant_type):
     else:
         return config["quant_bits"].get(quant_type)
 
+def _check_weight(module):
+    try:
+        return isinstance(module.weight.data, torch.Tensor)
+    except AttributeError:
+        return False
+
+class Post_Training_Quantize():
+    """Post_Training_Quantize contains two major function. The first is global scale quantization.
+    The second is mixed scale quantization.
+    """
+
+    def __init__(self, model, quant_bits=-1):
+        self.model = model
+        self.quant_bits = quant_bits
+        """
+        Parameters
+        ----------
+        model : torch.nn.module
+            Model to be quantized
+        quant_bits : int
+            bit-width for the quantization
+        """
+
+    def _quantize(self, real_val, scale, zero_point, bits):
+        transformed_val = zero_point + real_val / scale
+        qmin = 0
+        qmax = (1 << bits) - 1
+        clamped_val = torch.clamp(transformed_val, qmin, qmax)
+        quantized_val = torch.round(clamped_val)
+        return quantized_val
+
+    def _dequantize(self, quantized_val, scale, zero_point):
+        real_val = scale*(quantized_val-zero_point)
+        return real_val
+
+    def get_quantize_parameter(self, weight, quant_bits=None):
+        if quant_bits == None:
+            quant_bits = self.quant_bits
+        rmax = torch.max(weight)
+        rmin = torch.min(weight)
+        scale, zero_point = update_quantization_param(quant_bits, rmin, rmax)
+        return scale, zero_point
+
+    def quantize(self):
+        """
+        quantize real value to specific bit-width.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        torch.nn.module
```

Comment:
`torch.nn.module` -> `torch.nn.Module`. and explain the returned model a little bit